stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -298,6 +309,10 @@ public void setUsage(@Usage String usage) {
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_RECEIVER, mReceiver);
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_REDIRECT, mRedirect);
 
+        if (mType != null) {
```

Comment:
nit: could we use the same construct for both, null check and to put in the hash, either the `mType` variable or the accessor `getType()`